pact/Pact/Core/Builtin.hs

```diff
@@ -288,7 +288,7 @@ coreBuiltinToText = \case
   CoreDistinct -> "distinct"
   CoreFormat -> "format"
   CoreEnumerate -> "enumerate"
-  CoreEnumerateStepN -> "enumerate-step"
+  CoreEnumerateStepN -> ""
```

Comment:
@jmcardon I feel like going through all builtins (+ overloads) should be a separate ticket. wdyt?

Reply:
This changes hashes! Do not do this. We need to separate this out